master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,59 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``2.4.0`` ( ``2019-08-18`` )
+=====================================
+
+Bug fixes
+---------
+
+- Add an index to ``steps.started_at`` to boost expensive SQL queries.
```

Comment:
We should probably mention that this can take some time to upgrade on bigger instances (in a highlights paragraph)

Reply:
Done.